api/src/main/java/org/pmiops/workbench/db/dao/UserRecentResourceServiceImpl.java

```diff
@@ -100,6 +119,23 @@ private DbUserRecentResource makeUserRecentResource(
     return resource;
   }
 
+  private void updateUserRecentResourceEntry(
```

Comment:
this name is no longer correct

Reply:
I prefer this name as it is just recent resource (which includes access and modified)